mods/cnc/maps/funpark01/scj01ea.lua

```diff
@@ -84,7 +84,7 @@ WorldLoaded = function()
 	Trigger.OnIdle(DinoTrex, DinoTrex.Hunt)
 
 	ReinforceWithLandingCraft(RifleReinforcments, SeaEntryA.Location, BeachReinforceA.Location, BeachReinforceA.Location)
-	Trigger.AfterDelay(DateTime.Seconds(1), function() InitialUnitsArrived = true end)
+	Trigger.AfterDelay(DateTime.Seconds(3), function() InitialUnitsArrived = true end)
```

Comment:
The "regression" was introduced by 404086370774298a338027ef582a96e9a31fdcbf, which appears to have fixed an issue that was causing the landing craft reinforcement to insert its passengers into the world too early.  Failing 1-second was the technically correct behaviour, because the player *doesn't* have any of the required units (in the world) at this point in the mission.

Of course this means that we will now need to check all the other missions that use a similar script pattern.